core/api/transport/impl/http/http_listener_impl.cpp

```diff
@@ -11,12 +11,24 @@
 #include "api/transport/impl/http/http_session.hpp"
 
 namespace kagome::api {
-  HttpListenerImpl::HttpListenerImpl(std::shared_ptr<Context> context,
-                                     const Configuration &configuration,
-                                     SessionImpl::Configuration session_config)
+  HttpListenerImpl::HttpListenerImpl(
+      std::shared_ptr<Context> context,
+      const Configuration &configuration,
+      SessionImpl::Configuration session_config) try
       : context_(std::move(context)),
         acceptor_(*context_, configuration.endpoint),
-        session_config_{session_config} {}
+        session_config_ {
+    session_config
+  }
+  {}
+  catch (boost::wrapexcept<boost::system::system_error> &e) {
+    Logger logger = common::createLogger("RPC_HTTP_Listener");
```

Comment:
probably better create logger beforehand

Reply:
Not, we can't, because this in constructor, especially exception handler of constructor. There can use only static member. Or local variable, of course.